webapp/src/pluginConstants/form.ts

```diff
@@ -263,3 +318,76 @@ export const subscriptionFilterEventTypeReposOptions = () => {
     options.push(filterLabelValuePairAll);
     return options;
 };
+
+// Repos subscription filters
+export const mergeResultOptons: LabelValuePair[] = [
+    {
+        value: 'Succeeded',
+        label: 'Merge successful',
+    },
+    {
+        value: 'Unsuccessful',
+        label: 'Merge Unsucessful - Reason: Any',
```

Comment:
```suggestion
        label: 'Merge Unsuccessful - Reason: Any',
```
Fix this everywhere.